src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js

```diff
@@ -51,11 +51,13 @@ Behaviour.register({"SELECT.staticCombobox": function(e) {
     
     /*
      * Changes from the original 3:
-     *   Original behavior: Show candidates that matches the current incomplete input.
-     *   Changed behavior : Show all the candidates, not concerning with the current incomplete input.
+     *   Original behavior: Show candidates that start with the current incomplete input.

```

Comment:
@ikedam The original behaviour would return all items regardless of the input box, other than that it's fine.

Reply:
In this context, the original one means `<f:combobox>`.

Reply:
oh, ok then it makes sense.